cmd/podman/utils.go

```diff
@@ -73,3 +73,12 @@ func checkIfFileExists(name string) bool {
 	}
 	return !file.IsDir()
 }
+
+// Check if a file is or is not a directory
+func fileIsDir(name string) bool {
+	file, err := os.Stat(name)
+	if os.IsNotExist(err) {
```

Comment:
I think we need to return and check the error as well. We could face permission errors in which case `file == nil` which could lead to a nil dereference in line 83.